Wikipedia/Code/WMFExploreCollectionViewController.m

```diff
@@ -569,6 +568,7 @@ - (void)traitCollectionDidChange:(nullable UITraitCollection *)previousTraitColl
     [self resetLayoutCache];
     [super traitCollectionDidChange:previousTraitCollection];
     [self registerForPreviewingIfAvailable];
+    [self sizeNotificationHeader];
```

Comment:
It may be outside the scope of this ticket but I noticed the notification header also needs to be resized when the trait collection changes. 

Also, WMFExploreCollectionViewController doesn't properly resize the notification header following orientation changes which leads to text getting clipped or disappearing altogether. This might be a simulator only issue.